lib/rubocop/rspec/config_formatter.rb

```diff
@@ -4,15 +4,15 @@ module RuboCop
   module RSpec
     # Builds a YAML config file from two config hashes
     class ConfigFormatter
-      NAMESPACE = 'RSpec'.freeze
+      NAMESPACES = ['RSpec'.freeze, 'FactoryGirl'.freeze].freeze
 
       def initialize(config, descriptions)
         @config       = config
         @descriptions = descriptions
       end
 
       def dump
-        YAML.dump(unified_config).gsub(/^#{NAMESPACE}/, "\n#{NAMESPACE}")
+        YAML.dump(unified_config).gsub(/^(#{NAMESPACES.join('|')})/, "\n\\1")
```

Comment:
Cool @backus!
<details>
  <summary> Looks like work in the same way.</summary>

```ruby
simple =  /^(#{["a","b"].join("|")})/ # => /^(a|b)/
with_union = /^(#{Regexp.union("a","b")})/ # => /^((?-mix:a|b))/
puts "a first \nb second".gsub(simple, "\n\\1")

a first

b second
=> nil
puts "a first \nb second".gsub(with_union, "\n\\1")

a first

b second
=> nil
```
</details>